src/main/java/config/WebConfig.java

```diff
@@ -0,0 +1,24 @@
+package config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"controllers"})
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver resolver() {
```

Comment:
if you don't use jsp view pages, maybe you shouldn't add this method here and corresponding dependencies in pom.xml

Reply:
yes, but in case we don't have user with a corresponding id, thought somehow to redirect him to jsp